logrecorder/logrecorder-assertions/src/main/kotlin/info/novatec/testit/logrecorder/assertion/ContainsExactly.kt

```diff
@@ -0,0 +1,47 @@
+package info.novatec.testit.logrecorder.assertion
+
+import info.novatec.testit.logrecorder.api.LogEntry
+
+@DslContext
+class ContainsExactly : AbstractAssertionBlock() {
+
+    /**
+     * This matcher can be used to skip log messages, that are not of any interest.
+     * It will match any massage with any log level.
+     */
+    fun something() = any()
```

Comment:
`anything` sounds better IMO. Both mockito and assert libs that I know of use some variation of "any(thing)" if the shape and content of a value doesn't matter.